res/css/views/voip/_VideoFeed.scss

```
@@ -19,6 +19,8 @@ limitations under the License.
     height: 100%;
     background-color: #000;
     z-index: 50;
+    margin-top: auto;
+    margin-bottom: auto;
```

I believe two new CSS properties emerged fairly recently, [`object-fit`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) and [`object-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position). Have you tried using them in that context?

Given that `mx_VideoFeed_remote` is covering 100% of the available space of the parent it looks like it would be a good fit and would probably convey intent a bit more easily

```suggestion
    object-fit: contain;
    object-position: center;
```

---

I have to say that I'm not familiar with `object-fit` and `object-position`. I won't have time to check the results today, but I can check it tomorrow.